tests/php/tests/includes/test_class.wp-job-manager-usage-tracking-data.php

```diff
@@ -53,6 +53,20 @@ class WP_Test_WP_Job_Manager_Usage_Tracking_Data extends WPJM_BaseTest {
 	 */
 	private $publish;
 
+	/**
+	 * Set up for tests.
+	 */
+	public function setUp() {
+		parent::setUp();
+
+		// Ensure job categories and types are enabled.
+		update_option( 'job_manager_enable_categories', 1 );
+		update_option( 'job_manager_enable_types', 1 );
+		unregister_post_type( 'job_listing' );
```

Comment:
A more appropriate place for the unregistration to happen would be inside a `tearDown` function, since it's essentially undoing something created earlier.

Reply:
It needs to happen before `$post_type_instance->register_post_types();` because that doesn't update the post types (based on the options above) if it has already been registered.

Reply:
It ran OK for me when I put it in a `teardown`. 🤷‍♀️